finger-exercises-react/react/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,26 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...theArgs) {
+  if (theArgs.length === 0) {
+    return undefined;
+  } else if (isArray(theArgs[0])) {
+    return Math.min(...theArgs[0]);
+  }
+  return Math.min(...theArgs);
 }
 
-export function copy() {
+export function copy(obj) {
+  return isArray(obj) ? [...obj] : { ...obj };
+}
+
+export function reverseMerge(obj1, obj2) {
+  const newArray = [...obj2, ...obj1];
+  return newArray;
```

Comment:
You don't need the const, just return the array.
It's not wrong, it's just a suggestion.